org/istio.yaml

```diff
@@ -501,6 +503,7 @@ orgs:
         - SataQiu
         - sbezverk
         - sdake
+        - stevenctl
```

Comment:
@stevenctl sorry, I gave you bad advice.  Could you remove this line for now?  You should sign up to be a member, but not yet a maintainer, though I'm sure that'll change quickly